src/server/ua_server.c

```diff
@@ -128,6 +141,15 @@ UA_Server_getNamespaceByName(UA_Server *server, const UA_String namespaceUri,
     return res;
 }
 
+UA_StatusCode
+UA_Server_getNamespaceByIndex(UA_Server *server, const size_t namespaceIndex,
```

Comment:
The signature of this new method looks like it should be in the external api (include/open62541/server.h).
However it isn't added there or am i missing something?